### PR TITLE
User can see articles on screen

### PR DESCRIPTION
**Pull request by LauraRe**
#### Description
Changes proposed in this pull request:
* setup mobile client
* display articles on homescreen

#### What I have learned working on this feature:
* react-native basics
* how to send a get request to backend with react-native

#### Screenshots:
image

### CODE REVIEW

File: Services/ArticlesApiService.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+const url = `http://10.0.2.2:3000`; // works only for Android Emulator
+
+export const GetArticles = async () => {
+  let response = await axios.get(url + "/api/v1/articles");
+  const articles = response.data.articles;
+  return articles
+
+  try {
+  } catch (error) {
+    console.error(error);
```
Comment (Joberg95) [inline Services/ArticlesApiService.js]: Did we still need these? Since it worked with not having the lines 7-9 inside the "Try {}" at line 11

Comment (LauraRe) [reply Services/ArticlesApiService.js]: removed

File: Screens/Home/HomeScreen.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  renderArticles({ item }) {
+    const article = item;
+    return (
+      <View>
+        <Image
+          style={{ width: 100, height: 100 }}
+          source={{ uri: article.image }}
+        />
+        <Text>{article.category}</Text>
+        <Text>{article.title}</Text>
```
Comment (oliverochman) [inline Screens/Home/HomeScreen.js]: I would make this a header

File: Screens/Home/HomeScreen.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+  renderArticles({ item }) {
+    const article = item;
+    return (
+      <View>
+        <Image
+          style={{ width: 100, height: 100 }}
+          source={{ uri: article.image }}
+        />
+        <Text>{article.category}</Text>
```
Comment (oliverochman) [inline Screens/Home/HomeScreen.js]: In your LOFIs, do you have the category above the title of the article?

File: Services/ArticlesApiService.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import axios from "axios";
+
+// const url = `http://192.168.1.178:3000`;  // whenever we want to make api calls to localhost we have to use the ip address not the keyword `localhost` since that can result in a network error.
```
Comment (oliverochman) [inline Services/ArticlesApiService.js]: why do you need to keep this commented out code?

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: Screens/Home/HomeScreen.js (hunk 3 above)
Comment (Joberg95) [reply Screens/Home/HomeScreen.js]: No the category should be "invisible" and only be shown in the dropdown menu we are implementing later on.

File: Screens/Home/HomeScreen.js (hunk 2 above)
Comment (Joberg95) [reply Screens/Home/HomeScreen.js]: Done

File: Services/ArticlesApiService.js (hunk 4 above)
Comment (Joberg95) [reply Services/ArticlesApiService.js]: Removed

**Pull request merged**